Gruntfile.js

```diff
@@ -157,19 +231,87 @@ module.exports = function(grunt) {
             'ui-common': {
                 src: 'build/test-coverage/lcov/**/*.info',
             }
-        }
+        },
+        // file concatenation
+        concat: {
+            options: {
+                banner: cssFileHeader,
+            },
+            // css for the "page" template
+            pageCss: {
+                src: pageCssFiles,
+                dest: kbaseStaticDir + 'css/page_concat.css',
+                nonull: true,
+            },
+            // a single file containing all the KBase css!
+            allCss: {
+                src: allCssFiles,
+                dest: kbaseStaticDir + 'css/all_concat.css',
+                nonull: true,
+            }
```

Comment:
Concatenate all the css files in the `static/kbase/css` and `static/custom/` (minus any `*_concat.css` files!) into `all_concat.css`.